frontend/src/modules/EditZing/Components/NotesModal.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import {
+  Modal,
+  Box,
+  Button,
+  Typography,
+  OutlinedInput,
+  IconButton,
+} from '@mui/material'
+import LoadingButton from '@mui/lab/LoadingButton'
+import SaveIcon from '@mui/icons-material/Save'
+import CloseIcon from '@mui/icons-material/Close'
+
+import { API_ROOT } from '@core/Constants'
+import axios from 'axios'
+
+const NotesModal = (props: any) => {
+  const { courseId } = useParams<{ courseId: string }>()
+  const { open, student, handleClose, setSaved, setNotSaved } = props
```

Comment:
Props should be typed. Also suggest deconstructing the props object directly in the function parameters like other components

Reply:
Doned